pkg/apis/application/v1alpha1/types.go

```diff
@@ -1690,7 +1690,14 @@ func (p *AppProject) normalizePolicy(policy string) string {
 // OrphanedResourcesMonitorSettings holds settings of orphaned resources monitoring
 type OrphanedResourcesMonitorSettings struct {
 	// Warn indicates if warning condition should be created for apps which have orphaned resources
-	Warn *bool `json:"warn,omitempty" protobuf:"bytes,1,name=warn"`
+	Warn       *bool    `json:"warn,omitempty" protobuf:"bytes,1,name=warn"`
+	IgnoreList []Ignore `json:"ignoreList,omitempty" protobuf:"bytes,2,opt,name=ignoreList"`
```

Comment:
Please rename to just `Ignore`:
```suggestion
	Ignore []Ignore `json:"ignore,omitempty" protobuf:"bytes,2,opt,name=ignore"`
```